commit 0e8131bb32cf026c87baeacb7abf2a9bdbbc4953
Author: Andreas Schwab <schwab@redhat.com>
Date:   Sun May 8 21:48:03 2011 -0400

    Remove .UTF-8 suffix from locale names when it is the only supported codeset

diff --git a/localedata/SUPPORTED b/localedata/SUPPORTED
index d665961..e952c17 100644
--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -46,8 +46,8 @@ ar_TN.UTF-8/UTF-8 \
 ar_TN/ISO-8859-6 \
 ar_YE.UTF-8/UTF-8 \
 ar_YE/ISO-8859-6 \
-az_AZ.UTF-8/UTF-8 \
-as_IN.UTF-8/UTF-8 \
+az_AZ/UTF-8 \
+as_IN/UTF-8 \
 ast_ES.UTF-8/UTF-8 \
 ast_ES/ISO-8859-15 \
 be_BY.UTF-8/UTF-8 \
@@ -385,8 +385,8 @@ tr_CY/ISO-8859-9 \
 tr_TR.UTF-8/UTF-8 \
 tr_TR/ISO-8859-9 \
 ts_ZA/UTF-8 \
-tt_RU.UTF-8/UTF-8 \
-tt_RU.UTF-8@iqtelif/UTF-8 \
+tt_RU/UTF-8 \
+tt_RU@iqtelif/UTF-8 \
 ug_CN/UTF-8 \
 uk_UA.UTF-8/UTF-8 \
 uk_UA/KOI8-U \
